scripts/change-password.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
 
-curl --include --request PATCH http://localhost:3000/change-password/$ID \
-  --header "Authorization: Token token=$TOKEN" \
+API="http://localhost:4741"
```

Comment:
Why is everyone doing this? If you set a variable here, you can't set it from the command line. I'd prefer things like this be a comment, especially if it's formatted as a "usage" help-style comment.